src/job-feed/JobDetails.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {
+  Card, CardContent, CardActions, Typography, Button, Avatar,
+} from '@material-ui/core';
+import JobDetailsSection from './JobDetailsSection';
+import useStyles from './useStyles';
+
+function JobDetails({ job }) {
+  const classes = useStyles();
+
+  return (
+    <Card data-testid="job-details-data">
+      <CardContent>
+        <Typography variant="h3">
+          {job.name}
+        </Typography>
+        <div className={classes.companyLogo}>
+          <Avatar alt={job.company.name} src={job.company.contactInfo.avatarUrl} />
+          <Typography color="textSecondary">
+            {job.company.name}
+          </Typography>
+        </div>
+
+        <div className={classes.content}>
+          <JobDetailsSection
```

Comment:
I like how `JobDetailsSection` looks but providing it with such data isn't too great, you can extract it into a separate file as well (for both sections) Or make `JobDetailsSection` display both sections and accept a nested array instead